commercial/app/views/static/surveySimplePageSignup.scala.html

```diff
@@ -0,0 +1,63 @@
+@(StaticSurveySignupPage: controllers.commercial.StaticSurveySignupPage)(implicit request: RequestHeader)
+
+@import conf.switches.Switches._
+@import common.InlineJs
+@import templates.inlineJS.blocking.js._
+
+<!DOCTYPE html>
+<html id="js-context" class="js-off is-not-modern id--signed-out" data-page-path="@request.path">
+
+<head>
+    @fragments.head(StaticSurveySignupPage, None, Html(""))
+</head>
+    <body id="top" itemscope itemtype="http://schema.org/WebPage">
+
+        <a class="u-h skip" href="#maincontent" data-link-name="skip : main content">Skip to main content</a>
+
+        @fragments.header(StaticSurveySignupPage)
+
+        <div class="l-side-margins">
+
+            <div class="live-events-survey-wrapper" data-link-name="live events survey page">
+
+                <div class="live-events-survey-wrapper-inner">
+
+                    @fragments.inlineSvg("survey-thumbsup", "commercial")
+
+                    <h1>You're ahead of your time.</h1>
+
+                    <p class="standfirst">Sorry, but this feature isn't available yet. This is part of a test to gauge interest in new and upcoming features. So thank you, you have just helped improve the user experience of our site.</p>
```

Comment:
The decision from product owner was not to get user emails this time. It wasn't that significant number of users last time we did this.